Gemfile

```diff
@@ -41,6 +41,10 @@ gem 'friendly_id', '~> 5.1'
 
 gem 'vemv', github: 'vemv/vemv', ref: '2df6eeed573bb52c5fc02fc60e1e5e75988b66df'
 
+gem "haml-rails", "~> 0.9"
+gem 'money-rails', '~>1'
+gem 'stripe', github: 'stripe/stripe-ruby', ref: 'cb841602fad553f8495fae05940fef975b85449e'
```

Comment:
no, do it like this:

* install it without version
* see the version in the new Gemfile.lock
* update Gemfile
* run `bundle install` again (Gemfile.lock will be slightly different now)